cloud_info_provider/providers/ssl_utils.py

```diff
@@ -51,7 +51,7 @@ def get_endpoint_ca_information(endpoint_url, insecure=False, cafile=None,
 
         client_ssl = SSL.Connection(ctx, client)
         client_ssl.set_connect_state()
-        client_ssl.set_tlsext_host_name(host)
+        client_ssl.set_tlsext_host_name(host.encode('utf-8'))
```

Comment:
I'm a bit uncertain if this should still work fine on py2.7
Some testing would be good

Reply:
Would it be possible to use Travis for this?
Or just a simple test in a python CLI would be enough?

Reply:
the doubt was more if encoding a string in python2.7 would work, I had some issues in the past with some xml and encoding, but this one seems to work (tested it already in the fedcloud-catchall-ops). 

So I think this could be considered as resolved (but feel free to provide some unittesting for this file :wink: )

Reply:
OK, so let's merge.